openshift/release/knative-eventing-kafka-distributed-channel-ci.yaml

```diff
@@ -666,6 +654,18 @@ spec:
           name: kafka-webhook
           namespace: knative-eventing
 ---
```

Comment:
I will remove the bits for the distributed channel... we aer not using it at all